tools/snippet_shell.py

```diff
@@ -45,24 +45,24 @@ def _start_services(self, console_env):
         console_env['s'] = self._ad.snippet
 
     def _get_banner(self, serial):
-        return """
-Connected to {}. Call methods against:
-    ad (android_device.AndroidDevice)
-    snippet or s (Snippet)
-    """.format(serial)
+        return """Connected to {}. Call methods against:
+                  ad (android_device.AndroidDevice)
+                  snippet or s (Snippet)
+               """.format(serial)
 
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
         description='Interactive client for Mobly code snippets.')
     parser.add_argument(
-        '-s', '--serial',
-        help=
-        'Device serial to connect to (if more than one device is connected)')
-    args, argv = parser.parse_known_args()
-    if len(argv) != 1:
-        print('ERROR: Snippet package not specified on command line',
-              file=sys.stderr)
-        sys.exit(1)
-    logging.basicConfig(level=logging.INFO)
-    SnippetShell(argv[0]).main(args.serial)
+        '-s',
+        '--serial',
+        help='Device serial to connect to (if more than one device is connected)'
+    )
+    parser.add_argument('package', metavar='PACKAGE_NAME', type=str, nargs=1,
+                        help='The pacakge name of the snippet to use.')
+    args = parser.parse_args()
+    logger.setup_test_logger('/tmp/logs/mobly_snippet_shell',
```

Comment:
It shows a nice [SnippetShell] prefix with timestamp in log lines, instead of DEBUG::Xxx:: xyz.
It also helps to have a file tracing what happened especially if people want to use logging in the shell and do stuff?
But we don't have to make this change. No strong preference.